crud/stats/operation.lua

```diff
@@ -26,4 +26,5 @@ return {
     COUNT = 'count',
     -- BORDERS identifies both `min` and `max`.
     BORDERS = 'borders',
+    READVIEW = 'readview',
```

Comment:
Btw, for now it doesn't used in any way. It would be nice to monitor readview open/close calls, but I think it's better to create a ticket and implement it in the future.